schema/release-schema.json

```
@@ -517,6 +517,21 @@
             "description": "The single amendment object has been deprecated in favour of including amendments in an amendments (plural) array.",
             "deprecatedVersion": "1.1"
           }
+        },
+        "finalStatus": {
+          "title": "Tender final status",
```

Same for the others.

```suggestion
          "title": "Final status",
```

---

Done in 47c5ab09ffab6d5a09a0b0660f32460875cd10e6